src/lib/utils.ts

```diff
@@ -65,6 +76,24 @@ export function $resolve(dep: any = []) {
         }
       });
     }
+    case RemotingContext:
+    case Request:
+    case Response:
+    case AccessToken:
+    case Headers: {
+      let ctx = findContextArgument(args);
+      if (!ctx) {
+        throw new Error('Failed to resolve context argument for this function');
+      }
+      return Promise.resolve(resolveFromContext(dep, ctx));
+    }
+  }
+  if (dep.startsWith('@')) {
+    let ctx = findContextArgument(args);
+    if (!ctx) {
+      throw new Error('Failed to resolve context argument for this function');
+    }
+    return Promise.resolve(resolveFromContext(dep, ctx));
   }
   if (dep[0] === '^') {
```

Comment:
Probably should use `.startsWith` here too for consistency/readability?

`String.prototype.startsWith` is supported in Node 4+ so can't really see an issue.

http://node.green/#ES2015-built-in-extensions-String-prototype-methods-String-prototype-startsWith